bridge/setu/listener/maticchain.go

```diff
@@ -62,14 +61,7 @@ func (ml *MaticChainListener) ProcessHeader(newHeader *types.Header) {
 		return
 	}
 
-	chainmanagerParams, err := util.GetChainmanagerParams(ml.cliCtx)
-	if err != nil {
-		ml.Logger.Error("Error fetching chain manager params", "error", err)
-		return
-	}
-
-	confirmationTime := chainmanagerParams.TxConfirmationTime
-	ml.sendTaskWithDelay("sendCheckpointToHeimdall", headerBytes, confirmationTime)
+	ml.sendTaskWithDelay("sendCheckpointToHeimdall", headerBytes, 0)
```

Comment:
Don't you think we should keep delay
otherwise, it will unnecessarily attempt to send checkpoint 
TimeConfirmation + BlockConfirmation check would be better

Reply:
We are replacing timeconfirmation with blockConfirmations. 
Added block confirmations check on processor side which process only confirmed blocks.